src/index.html

```diff
@@ -1,14 +1,37 @@
+
 <!doctype html>
+
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header"> <a href="#home" class="header__image"> <img src="./images/logo.png" alt="Moyo logo"> </a>
```

Comment:
apply code styling recommendations from a checklist all over the page
```suggestion
    <header class="header"> 
      <a href="#home" class="header__image"> 
        <img 
          src="./images/logo.png" 
          alt="Moyo logo"
        > 
      </a>
```